index.html

```diff
@@ -67,6 +121,57 @@
     // Submit the form
     $("#dlForm").submit();
   })
+
+  $("#plus").click(newDoc);
+  $("#minus").click(function() {
+    // TODO: replace this with something prettier
+    if (!confirm('OK to delete?')) {
+      return;
+    }
+    var n = $('.cur').next();
+    if (n.length === 0) {
+      n = $('.cur').prev();
+    }
+    $('.cur').remove();
+    delete localStorage[cur];
+    if (n.length>0) {
+      cur = n[0].id;
+      load(cur);
+    } else {
+      newDoc();
+    }
+  });
+
+  // Automatically run the transform after a keypress, throttled with a timer.
+  $("#in").keyup(function() {
+    if (state.timer) {
+      clearTimeout(state.timer);
+    }
+    state.timer = setTimeout(function(){
+      state.timer = null;
+      save();
```

Comment:
Your approach is fine, except that after you stop typing, the last bits don't get transformed, so you still need a setTimeout.  Iet's stay with my approach for now, and we can change it when we've got a better idea.